tests/components/control4/test_config_flow.py

```diff
@@ -25,10 +24,7 @@ def _get_mock_c4_account(
         "href": "https://apis.control4.com/account/v3/rest/accounts/000000",
         "name": "Name",
     },
-    getDirectorBearerToken={
-        "token": "token",
-        "token_expiration": datetime.datetime(2020, 7, 15, 13, 50, 15, 26940),
-    },
+    getDirectorBearerToken={"token": "token"},
```

Comment:
Thanks!
It is directly linked to the dependency bump so it is correct to have it in the same PR.